mlir/test/Conversion/VectorToLLVM/vector-to-llvm-interface.mlir

```diff
@@ -130,6 +130,52 @@ func.func @broadcast_vec1d_from_vec1d_scalable(%arg0: vector<[2]xf32>) -> vector
 
 // -----
 
+// CHECK-LABEL: @broadcast_scalar_0d
+// CHECK-SAME: %[[ELT:.*]]: f32
+func.func @broadcast_scalar_0d(%elt: f32) -> vector<f32> {
```

Comment:
[nit] To follow the existing convention. Similar suggestion for the other tests below. 
```suggestion
func.func @broadcast_vec0d_from_scalar(%elt: f32) -> vector<f32> {
```